components/ListSurfspotsItems.js

```diff
@@ -23,4 +28,11 @@ const StyledLi = styled.li`
   padding: 10px;
   font-weight: normal;
   border-bottom: 2px solid #699bf7;
+  display: flex;
+  align-items: center;
+`;
+
+const StyledImage = styled(Image)`
+  position: absolute;
+  right: 60px;
```

Comment:
Just for the discussion: 
the "right: 60px" is also not that good for a responsive design. maybe it is better to stick the icon to the country?

Reply:
But then it would move when i change the display width or? The goal is that the arrow is always at the right side :)

Reply:
thats okay I guess